src/components/Cart/CartSummary.js

```diff
@@ -0,0 +1,75 @@
+import "../Card/CartCard.css";
+import "./Cart.css";
+import { cartDetails } from "../../utils/cartServices/cartDetails";
+import rupeeIcon from "../../assets/rupee.svg";
+const CartSummary = (props) => {
+  const { totalAmount, totalPrice, discount, deliveryCharges } = cartDetails(
+    props.details
+  );
+  return (
+    <>
+      <div class="product-details sm-gutter">
+        <div class="title para-sm xs-gutter bold">Price Details</div>
+        <div class="price xs-gutter">
+          <div>Price ({props.details.length})</div>
+          <div>
+            <img
+              src={rupeeIcon}
+              alt="rupee"
+              className="img-xxxs xxxs-padding rupee-icon"
+            />
+            {totalPrice}
+          </div>
+        </div>
+        <div class="price xs-gutter">
+          <div>Discount</div>
+          <div>
+            <img
+              src={rupeeIcon}
+              alt="rupee"
+              className="img-xxxs xxxs-padding rupee-icon"
+            />
+            {discount}
+          </div>
+        </div>
+        <div class="price xs-gutter">
+          <div>Delivery Charges</div>
+          <div>
+            <img
+              src={rupeeIcon}
+              alt="rupee"
+              className="img-xxxs rupee-icon xxxs-padding"
+            />
+            {deliveryCharges}
+          </div>
+        </div>
+        <div class="price xs-gutter top-padding">
+          <div class="bold">Total Amount</div>
+          <div class="bold">
+            <img
```

Comment:
change to className

Reply:
made all the changes please review and approve